doc/UserGuide/InstallAndBasicUsage.rst

```diff
@@ -216,41 +202,55 @@ You can also select specific info about the job object, e.g. the application tha
     :end-before: # -- INSTALLANDBASICUSAGE JOBSAPP STOP
     :dedent: 8
 
-To check the ``stdout/stderr`` of a job, you can use a couple of methods:
+To check the ``stdout/stderr`` of a job, you can use the peek method
 
 .. code-block:: python
 
-    jobs(0).peek('stdout', 'more')
     j = jobs(0)
-    !more $j.outputdir/stdout
+    j.peek('stdout')
+
+
+Copy a job
+----------
+You can copy and old job, modify its attributes anbd then submit it as a new one
+
+.. code-block:: python
 
+    j = jobs(0).copy()
+    j.application.args = ['Hello from a copy']
+    j.name = 'Copy'
+    j.submit()
 
+    
 Job Monitoring
 --------------
 
 While Ganga is running in interactive mode, a background thread goes through all your active jobs and checks
 to see what state they are in. Generally, jobs will transition from new -> submitted -> running -> completed/failed.
 As described above, the `jobs` command will show you the state of your jobs in the Ganga repository.
 
-In addition to this monitoring, there is also a web GUI provided that can be started using the ``--webgui``
-option which gives a graphical representation of your repository. [NOTE THIS NEEDS TESTING AND MAY NOT WORK AT PRESENT!]
-
-
 Scripting and Batch Mode
 ------------------------
 
-You can put your ganga commands into a python script and then execute it from the Ganga prompt like this:
+You can put your ganga commands into a python script
 
-.. literalinclude:: ../../ganga/GangaCore/test/GPI/TutorialTests.py
-    :start-after: # -- INSTALLANDBASICUSAGE EXECFILE START
-    :end-before: # -- INSTALLANDBASICUSAGE EXECFILE STOP
-    :dedent: 8
+.. code-block:: bash
+
+    [centos7] ~ % cat >> myfile.py
+    j = Job()
+    j.submit()
+    ^D
```

Comment:
Is the `^D` intended?

Reply:
Yes, it is the EOF when creating a file.